README.md

```diff
@@ -46,6 +46,27 @@ access." You can now run `siac` in a separate command prompt to interact with
 siad. From here, you can send money, mine blocks, upload and download
 files, and advertise yourself as a host.
 
+Building From Source
+--------------------
+
+To build from source, [Go 1.6 must be installed](https://golang.org/doc/install)
+on the system. Then simply use `go get`:
+
+```
+go get -u github.com/NebulousLabs/Sia/...
+```
+
+This will download the Sia repo to your `$GOPATH/src` folder, and install the
+`siad` and `siac` binaries in your `$GOPATH/bin` folder.
+
+To stay up-to-date, run the previous `go get` command again. Alternatively, you
+can use the Makefile provided in this repo. Run `git pull origin master` to
+pull the latest changes, and `make release-std` to build the new binaries. You
```

Comment:
This is on Go 1.6.2 following the [standard install instructions](https://golang.org/doc/install). To write `/usr/local` on Ubuntu 14 server, you have to be root, so extracting the tar needs to run as sudo.

Then that file in the archive is not writable except by owner:

```
$ tar -ztvf go1.6.2.linux-amd64.tar.gz | grep pkg/linux_amd64/runtime/internal/sys.a
-rw-r--r-- 0/0            3026 2016-04-19 19:51 go/pkg/linux_amd64/runtime/internal/sys.a
```